src/include/sof/audio/crossover/crossover.h

```diff
@@ -142,14 +143,101 @@ static inline crossover_split crossover_find_split_func(int32_t num_sinks)
 	return crossover_split_fnmap[num_sinks - CROSSOVER_2WAY_NUM_SINKS];
 }
 
+static inline int32_t iir_df2t_2stage_biquads(struct iir_state_df2t *iir, int32_t x)
```

Comment:
You are right, it should not be the inline function.
The purpose of unrolling is to define different quant precisions between the first and the second biquad, because I no longer saturate the output of the first biquad to Q1.31.